app/javascript/src/components/TaxStatus.vue

```diff
@@ -17,43 +17,50 @@
     <div v-if="showTaxStatus">
       <div class="flex justify-between">
         <span class="w-1/3"></span>
-        <span class="w-1/3">Power points</span>
-        <span v-if="game.baseGame === 'imperial'" class="w-1/3">Tax chart position</span>
+        <span class="w-1/5">Flags</span>
+        <span class="w-1/5">Tax amount</span>
```

Comment:
I love changing `Tax chart position` to `Tax amount`. Much more direct, useful info. Can we change the classes on these to `"w-1/4"` so that the fractions all add up to `1`? (If you want to subdivide `1` differently, that's fine, like make the initial `span` wider for the nation name)

Reply:
Heh, also addressed in the later PRs